administrator-manual/en/webtop5.rst

```diff
@@ -511,6 +615,18 @@ Proceed as follows:
 
 5. the next login will show the new logo on the login page
 
+Change the public URL
+=====================
+
+A special prop has been added to modify the public URL.
+
+If you want to change URL from this: ``http://server.domain.local/webtop`` to: ``http://mail.publicdomain.com/webtop``
+
+execute these commands:
```

Comment:
change the syntax, use the `::` operator

Reply:
done